pgcli/pgclirc

```diff
@@ -121,7 +121,7 @@ prompt = '\u@\h:\d> '
 min_num_menu_lines = 4
 
 # Character used to left pad multi-line queries to match the prompt size.
-multiline_continuation_char = '.'
+multiline_continuation_char = ''
```

Comment:
Good idea to be copy-pastable. I use `Ctrl+select` to select region in vte terminals.

That's not obvious that empty char will produce space padded prompt. I thought that this would drop padding.